jetty9-base/src/main/jetty-base/etc/gae-web.xml

```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0"?>
 <!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 <Configure class="org.eclipse.jetty.webapp.WebAppContext">
-
+ 
   <!-- Allow start exceptions to propagate -->
   <Set name="throwUnavailableOnStartupException">true</Set>
 
+  <!-- Add listener to capture request context to be used by logging  -->
+  <Call name="addEventListener">
+    <Arg><New class="com.google.cloud.runtimes.jetty9.RequestContextScope"/></Arg>
```

Comment:
Can we add this listener conditionally based on an environment variable? For example, if the image is used outside of GCP, this won't make sense.